torch/_dynamo/eval_frame.py

```diff
@@ -521,6 +521,36 @@ def guard_export_print(guards):
 def export(
     f, *args, aten_graph=False, decomposition_table=None, tracing_mode="real", **kwargs
 ):
+    """
+    Export an input function f to a format that can be executed outside of PyTorch using the FX graph.
+
+    Args:
+        f (callable): A PyTorch function to be exported.
+
+        *args: Variable length argument list to be passed to the function f.
+
+        aten_graph (bool): If True, exports a graph with ATen operators.
+        If False, exports a graph with Python operators. Default is False.
+
+        decomposition_table (dict): A dictionary that maps operators to their decomposition functions.
```

Comment:
Commenting as a humble end user: Not sure here what a decomposition table concretely looks like beyond a dict and what's teh default if I don't pass anything

not sure what dual and real mean exactly in tracing mode

The return type last I checked maybe a week ago was a tuple with a GraphModule and Guards, if that changed recently ignore this line